transformer/external/executabletransformer.go

```diff
@@ -50,6 +52,13 @@ type ExecutableYamlConfig struct {
 	Container          environmenttypes.Container `yaml:"container,omitempty"`
 }
 
+const (
+	transformerInputDir           = "artifacts"
+	transformerContainerOutputDir = "/var/tmp/output"
+	transformerInputArtifactFile  = "artifacts.json"
+	transformerOutputFile         = "output.json"
```

Comment:
can we have some prefixes to these files like say `m2k`, so that it does not conflict with any existing files?

Reply:
done